docs/userguide/quickstart.rst

```diff
@@ -59,11 +59,11 @@ This is what your project would look like::
         setup.cfg
         mypackage/__init__.py
 
-Then, you need an installer, such as `pep517 <https://pypi.org/project/pep517/>`_
-which you can obtain via ``pip install pep517``. After downloading it, invoke
+Then, you need an installer, such as `PyPA build <https://pypa-build.readthedocs.io/en/latest/>`_
```

Comment:
I like the idea of linking the python-build docs but could you please add their RTD page to intersphinx in the `conf.py` instead?

Reply:
I am new to sphinx. I have never heard of conf.py or intersphinx. I am not familiar with the nuances of rst syntax. I don't know what a domain is. I can't find the documentation for the `std` domain anywhere. I have had to read a whole bunch of documentation to figure out what you want me to do))))))

done

Reply:
Oh, it's the same as just `:doc:`, but intersphinx still exposes it as `:std:doc:`, I guess to produce less conflicts...

Thanks for approaching this issue, anyway :)

Reply:
(sorry for the typo in my suggestion)